roles/kubernetes/node/defaults/main.yml

```diff
@@ -263,3 +263,8 @@ conntrack_modules:
 kubelet_tracing: false
 kubelet_tracing_endpoint: 0.0.0.0:4317
 kubelet_tracing_sampling_rate_per_million: 100
+
+# Whether pull images one at a time
+kubelet_serialize_image_pulls: true
+## The maximum number of image pulls in parallel. This option cannot be set if kubelet_serialize_image_pulls is true
+# kubelet_max_parallel_image_pulls: 5
```

Comment:
Could you define it in any case and have the logic to not add in the kubelet config if `kubelet_serialize_image_pulls` is false? Because right now you have an ansible error as this variable is not defined

Reply:
My carelessness, I'll use `{% if kubelet_max_parallel_image_pulls is defined and not kubelet_serialize_image_pulls %}` to determine in advance whether the option is defined

Reply:
Yes but in kubespray we like to avoid using is defined in general, so could you not comment this out and do the conditioning in the kubelet config only based on `kubelet_max_parallel_image_pulls`, does that make sense?

Reply:
Right, but when `kubelet_max_parallel_image_pulls` is defined and own a initial value with `kubelet_serialize_image_pulls: true`, it would cause kubelet be failed to start. And  `kubelet_max_parallel_image_pulls: true` is the default behaviour within kubelet, I don't think we need to change the default behaviour. https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration

Reply:
Good solution, I'll take your way.